app/src/main/java/tttTDD/Java/Board.java

```diff
@@ -24,8 +23,12 @@ public String getSpace(int location) {
         return isLocationValid(location) ? null : this.board[location];
     }
 
+    public String getBoard() {
```

Comment:
I think it might be more idiomatic in Java to rename this function and have it `@Override` the `toString` method. Can you rename it?

A benefit of overriding the toString method (as minor as it might be) is that if you were to ever to pass a `Board` object to System.out, it will call toString automatically 

```
Board board = new Board();
board.updateBoard(1, "X");
board.updateBoard(2, "O");

System.out.println(board); // dont have to call toString, if you dont want to
=> "XO3456789"

```